planning/behavior_path_planner/docs/behavior_path_planner_start_planner_design.md

```diff
@@ -82,8 +82,108 @@ PullOutPath --o PullOutPlannerBase
 
 ## Safety check with dynamic obstacles
 
+### **Basic concept of safety check against dynamic obstacles**
+
+This is based on the concept of RSS. For the logic used, refer to the link below.
+See [safety check feature explanation](../docs/behavior_path_planner_safety_check.md)
+
+### **Collision check performed range**
+
+A collision check with dynamic objects is primarily performed between the shift start point and end point. The range for safety check varies depending on the type of path generated, so it will be explained for each pattern.
+
+#### **Shift pull out**
+
+For the "shift pull out", safety verification starts at the beginning of the shift and ends at the shift's conclusion.
+
+#### **Geometric pull out**
+
+Since there's a stop at the midpoint during the shift, this becomes the endpoint for safety verification. After stopping, safety verification resumes.
+
+#### **Backward pull out start point search**
+
+During backward movement, no safety check is performed. Safety check begins at the point where the backward movement ends.
+
+![collision_check_range](../image/start_planner/collision_check_range.drawio.svg)
```

Comment:
will replace the image with rotated footprint one in the future?

Reply:
changed in b6a72abd7504de0d9672715ea5c6308ef00fffd7

Reply:
![image](https://github.com/autowarefoundation/autoware.universe/assets/32741405/0118ac22-2f59-4bcb-b8e4-0d50fa3a45a4)